pkg/sfu/streamtracker/streamtracker_dd.go

```diff
@@ -126,15 +129,26 @@ func (s *StreamTrackerDependencyDescriptor) SetPaused(paused bool) {
 		return
 	}
 	s.paused = paused
+
+	var notifyFns []func(status StreamStatus)
+	var notifyStatus StreamStatus
 	if !paused {
 		s.resetLocked()
+
+		notifyStatus = StreamStatusStopped
+		notifyFns = s.onStatusChanged[:]
```

Comment:
this slice still references the array's memory

Reply:
Was getting lazy there 🙂. Those are set early in the life cycle. So used the lazy approach, but will not be and change it 🙂